demo/app/client/js/patterns/styles.jsx

```diff
@@ -14,7 +14,7 @@ export let styles = {
     description: (
         <div>
             <p>First, a button is created using <span className="pre">paypal.Button.render()</span>, and rendered to the <span className="pre">#paypal-button-container</span> element.</p>
-            <p>Along with the other required options, we pass a <span className="pre">style</span> block with various options to customize the button. In this block I can specify the <span className="pre">size</span>, <span className="pre">color</span> and <span className="pre">shape</span> of the button</p>
```

Comment:
Can we also list `style: 'checkout' // checkout | credit` to the example code for this section?